awscrt/s3.py

```diff
@@ -589,15 +589,26 @@ def is_optimized_for_system():
     return _awscrt.s3_is_crt_s3_optimized_for_system()
 
 
+def get_optimized_platforms():
```

Comment:
I also considered the following names:
- s3_get_platforms_with_recommended_config (similar to C but no concept of config in Pythong)
- get_platforms_with_recommended_throughput_target() (Too verbose and we might add more things in future)

Reply:
Given that the other public function is named `is_optimized_for_system()`, this is fine

if we were going to name this differently, then the other should be named differently too